source/sass/views/youtube-regrets-shared.scss

```diff
@@ -11,8 +11,8 @@
         $youtube-regrets-dark-blue 26.46%,
         $black 57.83%,
         $black 100%
-    ),
-    linear-gradient(0deg, $black, $black);
+      ),
+      linear-gradient(0deg, $black, $black);
```

Comment:
**Question** This is strange. Is this because the values in the function are indented by 4 instead of 2 maybe?

Reply:
Yea its super weird, it actually did this when I ran prettier haha so i guess i'll leave as is for now